roles/horizon/tasks/logging.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: set up log rotation for horizon
+  logrotate: name=horizon path=/var/log/apache2/*.log
+  args:
+    options:
+      - daily
+      - missingok
+      - rotate 7
+      - compress
```

Comment:
Well, it handles it, but doesn't have the same settings. What if we change then name here from horizon to apache2? That way we modify what's there to fit our needs.